src/clr-angular/forms/styles/_checkbox.clarity.scss

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-control-container {
+    .clr-checkbox-wrapper {
+        @include form-flatten-validate-text();
+
+        position: relative;
```

Comment:
I am assuming we added this because if someone had to add a tooltip, the would add it outside of the `.clr-checkbox-wrapper` right?

Reply:
This looks like an error, removing it.